parser/parser.go

```diff
@@ -330,6 +330,9 @@ func (p *parser) parseExpression() (value Expression) {
 	switch p.tok {
 	case '+':
 		return p.parseOperator(value)
+	case '-':
+		p.errorf("couldn't parse expression with '-' operator: %s", p.scanner.String())
```

Comment:
I think we'd like to clarify a little why this isn't parseable, for example "using '-' to indicate subtraction is not permitted ('-' may only be used to indicate negative numbers)"

Reply:
Agree, maybe "subtraction not supported"?

Reply:
done